jenkins/buster-chromebooks.jpl

```diff
@@ -0,0 +1,24 @@
+@Library('kernelci') _
+import org.kernelci.debian.RootFS
+
+/* ----------------------------------------------------------------------------
+ * Jenkins parameters
+
+DOCKER_BASE
+  Dockerhub base address used for the build images
+
+*/
+
+def r = new RootFS()
+
+def config = ['name':"buster-chromebooks",
+              'arch_list':["armhf", "arm64", "amd64"],
+              'debian_release':"buster",
+              'extra_packages':"python3-minimal python3-unittest2 python3-numpy",
```

Comment:
What is the resulting uncompressed size of the ramdisk, compared to the plain`buster` rootfs?

Maybe some files in `/usr/share` or elsewhere could be removed with the `extra_files_remove` attribute to optimize the size.

Reply:
Actually the size is 30M, I suppose there are some files that we can remove but I need to look at. Is it close to a non-working rootfs because of the size?

Reply:
OK I think it's actually the compressed size of the ramdisk that matters from a bootloader point of view.  Each device type has a different maximum size available for the ramdisk, bigger ramdisks are more likely to fail especially as the kernel image becomes bigger.  Also, smaller ramdisk images take less time to download and use less data.

The plain buster `rootfs.cpio.gz` is 15M, so that's basically twice the size.  I bet there are a lot of files that aren't really needed.  We can build one rootfs on staging as-is and try it anyway, especially since we already know that there will be a few iterations with this rootfs to then include the test suite.